tests/xfstests/run.pm

```diff
@@ -57,6 +57,10 @@ my $KDUMP_DIR = '/opt/kdump';
 my $MAX_TIME = get_var('XFSTESTS_SUBTEST_MAXTIME') || 2400;
 my $FSTYPE = get_required_var('XFSTESTS');
 
+# Variables use for no heartbeat mode
+my $TIMEOUT_NOHB = get_var('XFSTESTS_TIMEOUT', 2000);
```

Comment:
We have some long names already, i.e. XFSTESTS_SUBTEST_MAXTIME
Why do we use this cryptic one then: TIMEOUT_NOHB?
I would prefer we expand it and we don't use HB abbreviation.

Reply:
+1, I'll expand that